src/components/EventCard.tsx

```diff
@@ -0,0 +1,288 @@
+                      rel='noreferrer'
+                    >
+                      <span className='sr-only'>LinkedIn</span>
+                      <BsLinkedin />
+                    </a>
+                  </>
+                ) : (
+                  <></>
+                )}
+              </p>
+            </div>
+          </div>
+
+          {index < gridRows_o.length ? (
+            <>
+              <div className='flex items-center pt-3'>
+                {gridRows_o[index].thumbnail == '' ? (
+                  <>
+                    <div className='bg-blue-700 w-12 h-12 flex justify-center items-center rounded-full uppercase font-bold text-white'>
+                      {getShortName(gridRows_o[index].name)}
+                    </div>
+                  </>
+                ) : (
+                  <>
+                    <img
+                      className='object-cover w-32 h-32 mb-4 rounded-full shadow-md'
+                      src={gridRows_o[index].thumbnail}
+                      alt='avatar'
+                    />
+                  </>
+                )}
+                <div className='ml-4'>
+                  <p className='font-bold'>{gridRows_o[index].name}</p>
+                  <p className='text-sm text-gray-700 mt-1'>
+                    {gridRows_o[index].role} @ {gridRows_o[index].company}
+                    {gridRows_o[index].linkedinUrl != '' ? (
+                      <>
+                        <a
+                          href={gridRows_o[index].linkedinUrl}
+                          className='text-slate-800 hover:text-slate-600 dark:text-slate-100 dark:hover:text-white'
+                          target='_blank'
+                          rel='noreferrer'
+                        >
+                          <span className='sr-only'>LinkedIn</span>
+                          <BsLinkedin />
+                        </a>
+                      </>
+                    ) : (
+                      <></>
+                    )}
+                  </p>
+                </div>
+              </div>
+            </>
+          ) : (
+            <></>
+          )}
+        </div>
+      );
+    });
+  };
+
+  return (
+    <section className='sm:my-10'>
+      <div className='mx-auto px-4 sm:px-6 lg:px-8'>
+        <div className='relative -mx-4 overflow-hidden bg-slate-300 py-16 px-4 dark:bg-slate-800 sm:px-6 md:mx-0 md:rounded-3xl md:px-16 xl:px-24'>
+          <div className='flex flex-col bg-white shadow-lg rounded-lg overflow-hidden'>
+            <div className='flex justify-between bg-gray-200 text-gray-700 text-lg px-6 py-4'>
+              <span>
+                <button
+                  className='border-gray-400 rounded-full px-6 py-1 border'
+                  onClick={() => (window.location.href = './../')}
+                >
+                  <HomeIcon className='block h-6 w-6' aria-hidden='true' />
+                </button>
+              </span>
```

Comment:
Here I guess we can have 2 options

1- remove this button and take advantage of the header logo
![image](https://github.com/latina-in-tech/latina-in-tech.github.io/assets/822471/bedfdff6-b629-488b-9c13-4b2098e75ef7)
2- keep this icon but use the `Link` component
```suggestion
              <span className='border-gray-400 rounded-full px-6 py-1  border'>
                <Link href='/'>
                  <HomeIcon className='w-[32px]' aria-hidden='true' />
                </Link>
              </span>
```
If we move (for some reason) this component to another folder, the relative redirect cannot work anymore